.github/workflows/cuda-gpu-ci.yml

```diff
@@ -13,6 +13,10 @@ jobs:
     runs-on:
       group: cuda-gpu-runner-group
     name: Run Array API unit tests
+    permissions:
+      pull-requests: write
+      issues: write
+      statuses: write
```

Comment:
These permissions are taken from the linting bot. I'm not sure why we need issues and pull requests permissions, but I assume Adrin worked it out correctly. The statuses permission is needed to set the status of a commit.